OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/EdxAssessmentWebView.java

```diff
@@ -6,6 +6,8 @@
 public class EdxAssessmentWebView extends EdxWebView {
     public EdxAssessmentWebView(Context context, AttributeSet attrs) {
         super(context, attrs);
-        getSettings().setSupportZoom(false);
+        getSettings().setSupportZoom(true);
```

Comment:
Allowing zoom support here will enabling zoom feature for all the webviews. so better to handle it for specific xblock.

Reply:
done.